### PR TITLE
delete unused files, change index to redirect, update README

### PR DESCRIPTION
**Pull request by JacksonBurns**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,138 +1,13 @@
...
+<!-- https://stackoverflow.com/a/27895590 -->
 <head>
-    <meta charset='utf-8'>
-    <meta http-equiv="X-UA-Compatible" content="chrome=1">
-    <meta name="description" content="Reaction Mechanism Generator : RMG organization&#39;s &quot;home&quot; website">
-
-    <link rel="stylesheet" type="text/css" media="screen" href="stylesheets/stylesheet.css">
-
-    <title>Reaction Mechanism Generator</title>
+    <meta http-equiv='refresh' content='0; URL=http://example.com/'>
```
Comment (rwest) [inline index.html]: target URL?

Comment (JacksonBurns) [reply index.html]: Going to redirect to https://github.com/ReactionMechanismGenerator/RMG-Py just testing for now

**Comment by JacksonBurns** [comment]
Confirmed that this works by changing the github pages to build off of this branch, and then using multiple browsers to navigate to reactionmechanismgenerator.github.io which then successfully redirected to reactionmechanismgenerator.github.io/rmg-py

Going to change the pages build back to main, merge this branch, and delete this branch.

**Comment by JacksonBurns** [comment]
Confirmed with @jonwzheng offline as well, who is going to follow up with @mliu49 just to double-check that we haven't torn the roof off this thing entirely.

**Pull request merged**